src/Cake.Core.Tests/VerifyConfig.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Runtime.CompilerServices;
+using VerifyTests;
+using VerifyXunit;
+
+namespace Cake.Core.Tests
+{
+    public static class VerifyConfig
+    {
+        [ModuleInitializer]
+        public static void Init()
+        {
+            Verifier.DerivePathInfo(Expectations.Initialize);
```

Comment:
if u want to have `.verified.cake` as snapshot files then `EmptyFiles.FileExtensions.AddTextExtension("cake");` will make it easier